documentation/library_guide/parallel_api/execution_policies.rst

```diff
@@ -122,12 +122,12 @@ and ``using namespace sycl;`` directives when referring to policy classes and fu
 
 .. code:: cpp
 
-  auto policy_b = device_policy<class PolicyB> {device{gpu_selector{}}};
```

Comment:
```gpu_selector``` is deprecated: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:device-selector